GraphQL/Services/PersonPartObjectGraphType.cs

```diff
@@ -0,0 +1,43 @@
+﻿using GraphQL.Types;

```

Comment:
A `person` értéke mindig `null`:
![image](https://user-images.githubusercontent.com/1976647/131266045-850cf816-9256-4a9e-90b3-e77a3cc9d5d4.png)

És néhányuknál ilyen hiba van:
![image](https://user-images.githubusercontent.com/1976647/131266064-2a2c3c6d-3d04-40e1-ba48-7249a7efb4d8.png)

Reply:
A null az normális, amikor egy field exceptiont dob akkor null értéket kell visszaadnia.
Viszont ez szerintem valami upstream bug lesz, mert ha megfordítod a `biography` és a `person` sorrendjét akkor működik. 

Tehát így nálam nincs egy exception se:

```graphql
query MyQuery {
  personPage {
    person {
      biography
      birthDateUtc
      handedness
      name
    }
    biography
    displayText
  }
}
```

Reply:
Hogy fordítod meg?

Reply:
Kivágod a a tetejéről és beilleszted alá:
https://user-images.githubusercontent.com/4007293/131267947-43e13513-5337-42b3-ad91-d55197d115da.mp4

Más típusként resolve-olódik. Ha elől van a biography:
![image](https://user-images.githubusercontent.com/4007293/131268015-122c5eb6-2836-49ec-8ba2-c415c1ac2054.png)

Ha elöl van a person:
![image](https://user-images.githubusercontent.com/4007293/131267980-9e4f650a-d857-4fc9-9947-1f6bafd7cf7c.png)

Reply:
Ja, hogy így érted. Akkor egyrészt írd ide le légyszi, hogy erre figyeljen a delikvens, másrészt légyszi nyiss róla OC issue-t és linkeld be ide.

Reply:
Gondoltam rá, csak azt hittem ezt mergelni kéne, hogy hivatkozhassak rá a bugban. De tulajdonképpen nem kell, be tudom linkelni ezt a branchet. 

Íme: https://github.com/OrchardCMS/OrchardCore/issues/10181